README.md

```diff
@@ -72,6 +72,7 @@ Short option|Long option|Description
 (Not available)|`--singleplayer`|Start singleplayer game, useful for save editing, using/testing DXVK in singleplayer, etc.
 (Not available)|`--use-wined3d`|Use OpenGL-based D3D11 instead of DXVK when using Proton
 (Not available)|`--wine-desktop SIZE`|Use Wine desktop, work around missing TruckerMP overlay after tabbing out using DXVK, mouse clicking won't work in other GUI apps while the game is running, SIZE must be 'WIDTHxHEIGHT' format (e.g. 1920x1080)
+(Not available)|`--native-steam-dir`|Choose native Steam installation, useful only if you use Proton and you don't save account credentials on this computer [Default: `auto`]
```

Comment:
You may haven't noticed this but I have sorted the arguments alphabetically.
May be worth to rename both so they can group together.

Same in the argparser.